net-firewall/fwknop/fwknop-2.6.10.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
How about going to EAPI=7?

Reply:
Good call, in fact I had switched to EAPI=7 in my local overlay version and everything seemed fine.  Done.